eng/pipelines/ci.yaml

```diff
@@ -63,7 +63,6 @@ jobs:
         inputs:
           pwsh: true
           filePath: Generate.ps1
-          arguments: '-Parallelization 1'
```

Comment:
Are we sure the parallelization is stable enough for CI.

I do prefer longer time in CI, than CI not stable.

Reply:
This change should get us both faster and more stable CI 😄 

The reason it became unstable with Spotless being ran in parallel is that there could be multiple threads trying to download the Maven packages for Spotless and they could end up locking files against one another.

Now, all the code needed to run code formatting is bundled into the JAR, making it thread-safe.